testground/README.md

```
@@ -53,12 +78,253 @@ nix run .#stateless-testcase -- gen /tmp/data/out \
+cd /tmp/data
+echo 'FROM cronos-testground:latest
+ADD ./out /data' > Dockerfile.patched
+docker build -t cronos-testground:latest -f Dockerfile.patched .
+```
 
-Embed the data directory into the image, it produce a new image:
+### Method 3: Using Nix (patchimage command)
 
 ```bash
-$ nix run github:crypto-org-chain/cronos#stateless-testcase patchimage cronos-testground:latest /tmp/data/out
+$ nix run github:crypto-org-chain/cronos#stateless-testcase -- patchimage cronos-testground:latest /tmp/data/out
```

markdownlint-cli2 (0.21.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)